install.lua

```diff
@@ -52,6 +52,8 @@ for key, path in pairs(FILES) do
   if status then
     makeFile(path, response)
   else
+    fs.delete('github.rom')
+    fs.delete('github')
     printError(('Unable to download %s'):format(path))
   end
```

Comment:
should probably `break` here to stop more files being downloaded. Also, print the error before deleting, just in case deletion errors